app/boards/shields/zmk_uno/Kconfig.defconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_ZMK_UNO
+
+config ZMK_KEYBOARD_NAME
+    default "ZMK Uno"
+
+config ZMK_BACKLIGHT
+    select LED
+    select LED_GPIO
+
+config SHIELD_SSD1306_128X64
```

Comment:
A similar block for the 128x32 display would be helpful.

Reply:
Added.